.circleci/config.yml

```diff
@@ -83,87 +83,79 @@ jobs:
           ecrepouri: ${ECREPOURI_PSGA}
           awsregion: "eu-west-2"
           pathogen: "sars_cov_2"
-  build-ncov2019-artic-nf-illumina:
```

Comment:
git diff makes this nearly unreadable. 
It might be easier to check the final file directly.